datumaro/components/extractor_tfds.py

```diff
@@ -138,12 +139,11 @@ def __call__(
         item: DatasetItem,
         state: namespace,
     ) -> None:
-        if self.filename_feature_name:
-            filename = tfds_example[self.filename_feature_name].numpy().decode("UTF-8")
```

Comment:
Do we need to keep `filename_feature_name`? If not, we may lose the option, aren't we?

Reply:
For future reference, added code using `filename_feature_name` in https://github.com/openvinotoolkit/datumaro/pull/921/commits/a64b42f76c4757fa77bde32be69c1818d34a4683